infoschema/perfschema/tables_test.go

```diff
@@ -65,6 +66,11 @@ func (s *testTableSuite) TearDownSuite(c *C) {
 	s.store.Close()
 }
 
+func (s *testTableSuite) TestPredefinedTables(c *C) {
+	c.Assert(perfschema.IsPredefinedTable("EVENTS_statements_summary_by_digest"), IsTrue)
```

Comment:
Better put it into other suitable test case if there are only two lines for this function in the future.

Reply:
No other suitable test case in this file now.